Dockerfile

```diff
@@ -5,6 +5,6 @@ ADD . /go/src/willnorris.com/go/imageproxy
 RUN go get willnorris.com/go/imageproxy/cmd/imageproxy
 
 CMD []
-ENTRYPOINT /go/bin/imageproxy -addr 0.0.0.0:80
+ENTRYPOINT /go/bin/imageproxy -addr 0.0.0.0:80 -contentTypes=image/png,image/jpg,image/gif,image/webp,image/jpeg,image/pjpeg
```

Comment:
I see from the [README](https://github.com/willnorris/imageproxy/blob/master/README.md) the following:
> support for jpeg, png, webp (decode only), tiff, and gif image formats (including animated gifs)

Should we also support tiff by adding it to this list (`image/tiff`)?

Reply:
I can add tiff, and also bmp too actually... I think that facebook supports those image types, but I don't think any other network supports them

Reply:
👍